docs/source/example_vmec.rst

```diff
@@ -0,0 +1,318 @@
+Running VMEC 
+============
+
+In this tutorial, we describe how to run the VMEC equilibrium code. Other resources are the `STELLOPT wiki <https://princetonuniversity.github.io/STELLOPT/VMEC>`_ and the `VMEC++ documentation <https://arxiv.org/pdf/2502.04374>`_.
+
+Input parameters
+^^^^^^^^^^^^^^^^
+
+You can initialize this class either from a VMEC ``input.<extension>`` file or from a ``wout_<extension>.nc`` output file. If neither is provided, a default input file is used. When this class is initialized from an input file, it is possible to modify the input parameters and run the VMEC code. When this class is initialized from a ``wout`` file, all the data from the
```

Comment:
any chance prototypical input files for a vacuum and non-vacuum fields could be included here? could be useful.